content/applications/websites/ecommerce/managing_products/catalog.rst

```diff
@@ -0,0 +1,184 @@
+=======
+Catalog
+=======
+
+The eCommerce catalog is the equivalent of your physical store shelves: it allows customers to see
+what you have to offer. Clear categories, available options, sorting, and navigation threads help
+you structure it efficiently.
+
+Categorize the product catalog
+==============================
+
+In Odoo, there is a **specific category model** for your eCommerce. Using eCommerce categories for
+your products allows you to add a navigation menu on your eCommerce page. Visitors can then use it
+to view all products under the category they select. To do so, go to :menuselection:`Website app
```

Comment:
Suggestion: 
split before "to do so" (it could be in a separate paragraph anyway) and add an image to illustrate (not too large)?